task_reactredux/task1/src/reducers/index.js

```diff
@@ -2,7 +2,21 @@ import { combineReducers } from 'redux';
 import formReducer from './FormReducer';
 import passwordReducer from './PasswordReducer';
 import mailReducer from './MailReducer';
+import formSuccess from './FormSuccessReducer';
+import { reducer as reduxFormReducer } from 'redux-form';
+import { LOGIN_CLEAR_STATE } from '../actions/types';
 
-const reducer = combineReducers({form: formReducer, password: passwordReducer, mail: mailReducer  });
+const reducer = combineReducers({
+    formRed: formReducer, password: passwordReducer, mail: mailReducer, formSuc: formSuccess, form: reduxFormReducer.plugin ({
```

Comment:
please don't add abbreviations `formRed, formSuc`. Add full names, `formReducer, formSuccess`. Or even you could name last like `AuthorizationReducer`, it will be better